src/Networking/ParseBaseStation.cpp

```diff
@@ -139,7 +139,7 @@ bool ParseDrivePacket(json &message) {
   {
     return sendError("Drive targets not within bounds +/- 1.0");
   }
-  double MAX_X_VEL = 1.0; // m/s
+  double MAX_X_VEL = 0.5; // m/s
   double MAX_TH_VEL = 1.0; // rad/s
```

Comment:
If these are constants, we should probably move them to the beginning of the file